proxy-attestation-server/src/attestation/sgx.rs

```diff
@@ -256,6 +258,7 @@ pub fn msg3(body_string: String) -> ProxyAttestationServerResponder {
             })?
         };
 
+        // TODO: This function call needs to be reworked
```

Comment:
I assume it is for the future?

Reply:
Yes, the function isn't new to this change